java/src/jmri/jmrit/operations/router/Router.java

```diff
@@ -472,13 +484,16 @@ private boolean setCarDestinationTwoTrains(Car car, String trackType) {
                 addLine(_buildReport, SEVEN, BLANK_LINE);
                 addLine(_buildReport, SEVEN,
                         MessageFormat.format(Bundle.getMessage("RouterFoundTrack"),
-                                new Object[] { Track.getTrackTypeName(trackType), track.getLocation().getName(),
-                                        track.getName(), car.toString() }));
+                                new Object[]{Track.getTrackTypeName(trackType), track.getLocation().getName(),
+                                        track.getName(), car.toString()}));
             }
-            // test to see if there's a train that can deliver the car to its final location
+            // test to see if there's a train that can deliver the car to its
+            // final location
             testCar.setTrack(track);
             testCar.setDestination(car.getFinalDestination());
-            testCar.setDestinationTrack(car.getFinalDestinationTrack()); // can be null
+            testCar.setDestinationTrack(car.getFinalDestinationTrack()); // can
+                                                                         // be
+                                                                         // null
```

Comment:
Just checking. Is this change desired? There is a lot of changes in this PR that seems to be only a change in formatting, for example removing a space.

Reply:
Due to automatic formatting.  Not desired, but I didn't review all of the changes due to formatting.  Also lots of changes due to space formatting.  I had to recently rebuild my IDE, so that might explain the formatting changes.